README.md

```diff
@@ -1,11 +1,16 @@
 # Webnative App Template
 
-[![Built by FISSION](https://img.shields.io/badge/⌘-Built_by_FISSION-purple.svg)](https://fission.codes) [![Built by FISSION](https://img.shields.io/badge/webnative-v0.34.1-purple.svg)](https://github.com/fission-suite/webnative) [![Discord](https://img.shields.io/discord/478735028319158273.svg)](https://discord.gg/zAQBDEq) [![Discourse](https://img.shields.io/discourse/https/talk.fission.codes/topics)](https://talk.fission.codes) [![Netlify Status](https://api.netlify.com/api/v1/badges/7b7418ef-86eb-43c4-a668-0118568c7f46/deploy-status)](https://app.netlify.com/sites/webnative/deploys)
+[![Built by FISSION](https://img.shields.io/badge/⌘-Built_by_FISSION-purple.svg)](https://fission.codes) [![Built by FISSION](https://img.shields.io/badge/webnative-v0.34.1-purple.svg)](https://github.com/fission-suite/webnative) [![Discord](https://img.shields.io/discord/478735028319158273.svg)](https://discord.gg/zAQBDEq) [![Discourse](https://img.shields.io/discourse/https/talk.fission.codes/topics)](https://talk.fission.codes)
 
 ![Webnative UI Screenshot](static/webnative-ui.png)
 
 The Webnative App Template is a clone-and-go template for building a web application using Webnative, fast. Clone, customize, and deploy to have a running distributed app in mere minutes.
 
+<div style="background-color: #FEEB80; padding: 1px 20px; border-radius: 10px;">
```

Comment:
This is not displaying in the diff properly. It actually looks something like this:
![image](https://user-images.githubusercontent.com/27258/198682160-0bcbe24f-d554-450a-8b1e-ea79c5aa45da.png)

Reply:
Nope that's about right.

Reply:
The warning box isn't displaying for me when I view the rendered version here: https://github.com/webnative-examples/webnative-app-template/blob/449a28488fe21c6ea696b76684515407ba3a3da0/README.md